CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * In [filesets or path patterns](docs/filesets.md#file-patterns), glob matching
   is enabled by default. You can use `cwd:"path"` to match literal paths.
 
+* In the following commands, string patterns are now parsed as revset-like
+  expressions, and can be combined with logical operators: `jj bookmark list`,
```

Comment:
To reduce the risk of the reader interpreting "revset-like expression" as saying that these commands somehow support specifying revisions in these patterns, how about something like "string pattern arguments are now parsed the same way they are in revsets"?